### PR TITLE
Addition of parent container barcode and coord for extraction and sample QC

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

File: backend/fms_core/templates.py [hunk 1]
```diff
@@ -471,7 +471,10 @@
       ("SampleQC", "Sample Name", "name", "name"),
       ("SampleQC", "Sample Container Barcode", "container__barcode", "container_barcode"),
       ("SampleQC", "Sample Container Coord", "coordinate__name", "coordinates"),
-      ("SampleQC", "Current Volume (uL)", "volume", "volume"),],
+      ("SampleQC", "Sample Container Location Barcode", "container__location__barcode", "container_location_barcode"),
+      ("SampleQC", "Sample Container Location Coord", "container__coodinate__name", "container_location_coordinates"),
```
Comment (ckostiw) [inline backend/fms_core/templates.py:475]:
```suggestion
      ("SampleQC", "Sample Container Location Coord", "container__coordinate__name", "container_location_coordinates"),
```

File: backend/fms_core/templates.py [hunk 2]
```diff
@@ -497,7 +500,10 @@
       ("ExtractionTemplate", "Current Volume (uL)", "volume", "volume"),
       ("ExtractionTemplate", "Source Sample Name", "name", "name"),
       ("ExtractionTemplate", "Source Container Barcode", "container__barcode", "container_barcode"),
-      ("ExtractionTemplate", "Source Container Coord", "coordinate__name", "coordinates"),],
+      ("ExtractionTemplate", "Source Container Coord", "coordinate__name", "coordinates"),
+      ("ExtractionTemplate", "Source Container Location Barcode", "container__location__barcode", "container_location_barcode"),
+      ("ExtractionTemplate", "Source Container Location Coord", "container__coodinate__name", "container_location_coordinates"),
```
Comment (ckostiw) [inline backend/fms_core/templates.py:505]:
```suggestion
      ("ExtractionTemplate", "Source Container Location Coord", "container__coordinate__name", "container_location_coordinates"),
```

**Review by ckostiw — CHANGES_REQUESTED** [review]

File: backend/fms_core/static/submission_templates/Sample_extraction_v4_4_0.xlsx
Comment (ckostiw) [inline backend/fms_core/static/submission_templates/Sample_extraction_v4_4_0.xlsx:1]: The data validation on the Destination Container Coord column is broken (no popup).

Comment (ckostiw) [inline backend/fms_core/static/submission_templates/Sample_extraction_v4_4_0.xlsx:1]: We have "Source Container Location Barcode" for the parent source container, but "Destination Parent Container Barcode" for the destination container, so the column naming is inconsistent and confusing. Maybe "Destination Parent Container Barcode" should be changed to "Destination Container Location Barcode". Same goes for the parent destination coord.

**Review by ckostiw — CHANGES_REQUESTED** [review]

**Review by ckostiw — APPROVED** [review]

File: backend/fms_core/static/submission_templates/Sample_extraction_v4_4_0.xlsx
Comment (UlysseFG) [reply backend/fms_core/static/submission_templates/Sample_extraction_v4_4_0.xlsx:1]: I will use the parent naming all around ...

**Pull request merged**